CHANGES.md

```diff
@@ -1,9 +1,23 @@
 # Changes
 
+## edge-20.10.6
+
+This edge supersedes edge-20.10.5 as a release candidate for stable-2.9.0. It
+adds a new `linkerd.io/inject: ingress` annotation to support per-request
```

Comment:
Describing ingress mode as enabling "per-request routing" probably doesn't mean anything to anyone unfamiliar with the details of how the proxy works.  Perhaps we could instead describe this as a mode that allows ingresses to use service profiles and enables per-route metrics and traffic splits.

Reply:
good call, i just copied the language from the proxy's changelog